combinations/hash.tsv

```diff
@@ -266,7 +266,7 @@ star=2.7.10a,samtools=1.16.1,gawk=5.1.0
 python=2.7,pysam=0.15.2,flask=1.1.2,cython=0.29.14,numpy=1.16.6,scipy=1.2.1,matplotlib=2.2.5,mosek::mosek=8.0.60,future=0.18.2
 dnaio=0.8.1,pysam=0.19.0
 numpy=1.22.3,pandas=1.4.2,seaborn=0.11.2,deeptools=3.5.1,pysam=0.19.0,pyranges=0.0.115,dask=2022.4.1,upsetplot=0.6.0
-epytope=3.1.0,gawk=5.1.0,tcsh=6.20.00,perl=5.32.1
+python=3.7.12,epytope=3.3.0,gawk=5.1.0,tcsh=6.20.00,perl=5.32.1
```

Comment:
```suggestion
conda-forge::python=3.7.12,bioconda::epytope=3.3.0,conda-forge::gawk=5.1.0,conda-forge::tcsh=6.24.05,perl=5.32.1
```

Reply:
Could we try this @jonasscheid @christopher-mohr

Reply:
This is already what you get without `--strict-channel-priority`. Is the difference in the tensorflow packages? I see one coming from the `default` channel instead of the `conda-forge` channel.